Automattic/simplenote-android
# Update About Layout

### Fix
Update the ***About*** screen with the new layout.  See the screenshots below for illustration.

![update_about_layout_phone](https://user-images.githubusercontent.com/3827611/76906063-364f9600-6871-11ea-8aa5-50b70e65d284.png)

![update_about_layout_tablet](https://user-images.githubusercontent.com/3827611/76906070-38195980-6871-11ea-8fbb-b5e10620ba55.png)

The theme for the ***About*** screen was also updated so that the top app bar would show `blue` (`#2271b1`) rather than the default purple (`#bb86fc`) when viewing the recent apps on 
Android 8.1.0 (i.e. API 27) and below.  See the screenshots below for illustration.

![update_about_layout_phone_recent](https://user-images.githubusercontent.com/3827611/76906077-3c457700-6871-11ea-9fd7-3b971daf5728.png)

My only concern with these changes is the touch feedback background.  It isn't flush with the top or bottom dividers.  @SylvesterWilmott, do you think that's alright?  See the screenshot below for illustration.

<a href="https://user-images.githubusercontent.com/3827611/76906060-3354a580-6871-11ea-9170-e28f7a16c022.png"><img src="https://user-images.githubusercontent.com/3827611/76906060-3354a580-6871-11ea-9170-e28f7a16c022.png" width="300"></a>
![update_about_layout_phone_after_pressed]

### Test
1. Open navigation drawer.
2. Tap ***Settings*** item in navigation drawer.
3. Tap ***About Simplenote*** item under ***More Information*** section.
4. Notice layout as shown above.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
